### PR TITLE
Add windows build

### PR DESCRIPTION
**Pull request by 183amir**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,6 @@ source:
 
 build:
   number: 0
-  skip: true   # [win]
```
Comment (183amir) [inline recipe/meta.yaml]: @AliKhoda you still have not put the vc features here.
Please do so according this guide: https://conda-forge.org/docs/buildwin.html#vc-features

**Comment by 183amir** [comment]
@conda-forge-admin, please rerender

**Comment by 183amir** [comment]
@conda-forge-admin, please rerender

**Comment by 183amir** [comment]
@conda-forge-admin, please rerender

**Comment by AliKhoda** [comment]
please rerender

**Comment by AliKhoda** [comment]
@conda-forge-admin, please rerender

**Comment by 183amir** [comment]
looks like travis takes 3 days to build things. Since this pull request is changing the windows builds, I will merge this.

**Pull request merged**